### PR TITLE
cleaned docs-v2

### LINKED ISSUE #1586: Cleanup old docs workflows and remove references to "Common", "v1" and "v2"
We've now transferred the k8ssandra-operator docs contents and workflows to the k8ssandra-operator repository, and need to clean up any conflicting workflows in the k8ssandra repository.

That includes:

- `docs` and `docs-v2` workflows
- crd updates workflow

We also need to change the docs-v1 mentions of "Common" to become "k8ssandra-operator (current)" and "v1" to become "K8ssandra v1 (deprecated)". Occurrences of "v2" have to be removed.
The same changes have to be done in the k8ssandra-operator `main` branch.

### PR DESCRIPTION
**Pull request by RachanKaur**
**What this PR does**: Cleanup old docs

**Which issue(s) this PR fixes**:
Fixes #1586 

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: docs/config/development/config.toml [hunk 1]
```diff
@@ -3,18 +3,12 @@ url_latest_version = "http://localhost:1313/"
...
   version = "Common"
-  fullversion = "Common Information"
+  fullversion = "k8ssandra-operator"
   githubbranch = "docs"
   url = "http://localhost:1313"
 
 [[params.versions]]
   version = "v1"
-  fullversion = "k8ssandra-v1"
+  fullversion = "k8ssandra-v1 (deprecated)"
```
Comment (adejanovski) [inline docs/config/development/config.toml:12]:
```suggestion
  fullversion = "K8ssandra v1 (deprecated)"
```

File: docs/config/staging-v1/config.toml [hunk 2]
```diff
@@ -3,18 +3,13 @@ url_latest_version = "https://docs-staging-v1.k8ssandra.io"
...
   version = "Common"
-  fullversion = "Common Information"
+  fullversion = "k8ssandra-operator"
   githubbranch = "docs"
   url = "https://docs-staging.k8ssandra.io"
 
 [[params.versions]]
   version = "v1"
-  fullversion = "k8ssandra-v1"
+  fullversion = "k8ssandra-v1 (deprecated)"
```
Comment (adejanovski) [inline docs/config/staging-v1/config.toml:12]:
```suggestion
  fullversion = "K8ssandra v1 (deprecated)"
```

File: docs/config/staging/config.toml [hunk 3]
```diff
@@ -3,18 +3,13 @@ url_latest_version = "https://docs-staging.k8ssandra.io"
...
   version = "Common"
-  fullversion = "Common Information"
+  fullversion = "k8ssandra-operator"
   githubbranch = "docs"
   url = "https://docs-staging.k8ssandra.io"
 
 [[params.versions]]
   version = "v1"
-  fullversion = "k8ssandra-v1"
+  fullversion = "k8ssandra-v1 (deprecated)"
```
Comment (adejanovski) [inline docs/config/staging/config.toml:12]:
```suggestion
  fullversion = "K8ssandra v1 (deprecated)"
```

**Review by adejanovski — CHANGES_REQUESTED** [review]
Which branch did you use as base to create this PR?

The PR we want to push here will target `docs-v1` (which will be the only one that this repo will still publish). 
We will have to remove all publishing workflows in `docs` and `docs-v2` as well.

**Comment by adejanovski** [comment]
to be clear, this PR targets `main` which is not what we should target.
There's follow up work in k8ssandra-operator to apply the same changes (`Common` -> `k8ssandra-operator`, ...)

File: .github/workflows/update-crd-reference.yaml [hunk 4]
```diff
@@ -21,11 +21,6 @@ jobs:
           repository: k8ssandra/cass-operator
           ref: master
           path: cass-operator
```
[the remark this answers is not in the document]
Comment (RachanKaur) [reply .github/workflows/update-crd-reference.yaml:23]: okay

[the remark this answers is not in the document]
Comment (RachanKaur) [reply .github/workflows/update-crd-reference.yaml]: removed the file, do you want new crd workflow for v1 or leave it as it is deprecated?

**Comment by RachanKaur** [comment]
> 

This is the PR for follow up work in k8ssandra-operator https://github.com/k8ssandra/k8ssandra-operator/pull/893

**Pull request closed without merging**